metric2/meter/meter.go

```diff
@@ -0,0 +1,53 @@
+package meter
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric2/async"
+	"go.opentelemetry.io/otel/metric2/sync"
+)
+
+// MeterProvider supports creating named Meter instances, for
+// instrumenting an application containing multiple libraries of code.
+type MeterProvider interface {
+	Meter(instrumentationName string /*, opts ...MeterOption*/) Meter
+}
+
+// Meter is an instance of an OpenTelemetry metrics interface for an
+// individual named library of code.  This is the top-level entry
+// point for creating instruments.
+type Meter struct {
+}
+
+// Asynchronous provides access to an async.Meter for constructing
+// asynchronous metric instruments.
+func (m Meter) Asynchronous() async.Meter {
```

Comment:
I think this is a bit hard to understand the fact that there are 3 meters, I would reconsider this.